### ISSUE TITLE
[depth_from_video_in_the_wild]Bad depth  results on custom data.

### ISSUE THREAD

**Issue by poornimajd** [google-research/google-research#319]
Hello great work @gariel-google and team!
I tried training the model without any pretrained weights on about 5300 images for about 35 epochs and I got the following loss graph

But the result of depth inference is :
![0000001419](https://user-images.githubusercontent.com/31023599/87225164-b2359700-c3a8-11ea-91b4-f47502bd5e4e.png)
The result without training and directly infering on cityscapes pretrained weights provided in the repo is little better.
![0000001418](https://user-images.githubusercontent.com/31023599/87225199-e7da8000-c3a8-11ea-8922-ac5293e6f19b.png)
I took the inference code from struct2depth and replaced their model with depth_from_video_in_the_wild's model
When I plot the egomotion with the trained model, it looks ok.
Can I know what might have gone wrong while training?
Any suggestion is greatly appreciated.
Thanks.

**Comment by ikramabdel** [comment]
@poornimajd hi can you please share how you replaced the model in the inference code Struct2depth to get those results ? Thanks !

**Comment by poornimajd** [comment]
Hello @ikramabdel ,Its been a while since I did the inference,but I hope the following snippet works

`import numpy as np
import tensorflow as tf
from depth_from_video_in_the_wild import model
import cv2
import matplotlib.pyplot as plt


train_model = model.Model(img_height=None, img_width=None, batch_size=None, is_training=False)
sess = tf.Session()
saver = tf.train.Saver()
saver.restore(sess, './depth_from_video_in_the_wild/cityscapes_learned_intrinsics/model.ckpt')
image = cv2.imread('./depth_from_video_in_the_wild/rgb_image_93.jpgl.png')
inputs = cv2.resize(image, (416, 128))[np.newaxis,:]
depth = train_model.inference_depth(inputs, sess)

dd=(depth[0]*256)
cv2.imwrite('./depth_from_video_in_the_wild/depth.png',dd)

plt.figure(figsize=(20, 10))
plt.imshow(image)
plt.figure(figsize=(20, 10))
plt.imshow(1/depth[0,:,:,0], cmap='plasma')`

**Comment by ikramabdel** [comment]
> Hello @ikramabdel ,Its been a while since I did the inference,but I hope the following snippet works
> 
> `import` numpy as np
> import tensorflow as tf
> from depth_from_video_in_the_wild import model
> import cv2
> import matplotlib.pyplot as plt
> 
> train_model = model.Model(img_height=None, img_width=None, batch_size=None, is_training=False)
> sess = tf.Session()
> saver = tf.train.Saver()
> saver.restore(sess, './depth_from_video_in_the_wild/cityscapes_learned_intrinsics/model.ckpt')
> image = cv2.imread('./depth_from_video_in_the_wild/rgb_image_93.jpgl.png')
> inputs = cv2.resize(image, (416, 128))[np.newaxis,:]
> depth = train_model.inference_depth(inputs, sess)
> 
> dd=(depth[0]*256)
> cv2.imwrite('./depth_from_video_in_the_wild/depth.png',dd)
> 
> plt.figure(figsize=(20, 10))
> plt.imshow(image)
> plt.figure(figsize=(20, 10))
> plt.imshow(1/depth[0,:,:,0], cmap='plasma')

Thank for replying, how do you get a model.ckpt? i am loading model and the graph
